src/v2/view-builder/views/captcha/CaptchaView.js

```diff
@@ -23,6 +23,31 @@ const RECAPTCHAV2_URL =
 export default View.extend({
   className: 'captcha-view',
 
+  template: hbs`
+    {{#if isCaptchaConfigured}}\
+      <div id ="captcha-container"
+          class="{{class}}"
+          data-sitekey="{{sitekey}}"
+          data-callback="OktaSignInWidgetOnCaptchaSolved"
```

Comment:
Could 'OktaSignInWidgetOnCaptchaSolved` be defined as variable (just because this string has been used at 2 places)?

Reply:
Just to clarify, I'm assuming you're suggesting to use a template variable here?